backends/qualcomm/passes/utils.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Qualcomm Innovation Center, Inc.
+# All rights reserved
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+from executorch.backends.qualcomm.builders.utils import get_parameter
+from executorch.exir.dialects._ops import ops as exir_ops
+
+
+def get_quant_attrs(
+    edge_program: torch.export.ExportedProgram, quant_node: torch.fx.Node
+):
+    quant_attr_keys = [arg.name for arg in quant_node.target._schema.arguments][1:]
+    quant_attrs = dict.fromkeys(quant_attr_keys)
+
+    for i in range(1, len(quant_node.args)):
+        attr_n = quant_node.args[i]
+
+        value = attr_n
+        if type(attr_n) == torch.fx.node.Node:
+            # could be a commonly shared attribute between q & dq
+            if attr_n.target == exir_ops.edge.aten._to_copy.default:
```

Comment:
Why this special handing. Are some quant node's args are from copy? and why?

Reply:
This is also introduced by lifting behavior, the parameters in placeholder format will share the same scale and offset for the consecutive q and dq node.